packages/model/src/lab.ts

```diff
@@ -0,0 +1,4 @@
+export interface Lab {
+  id: string;
+  name?: string | null;
```

Comment:
It shoulnd't need the `null`as a type since it is optional

Reply:
I think we need it because of this
https://github.com/yldio/asap-hub/blob/master/packages/squidex/src/entities/common.ts#L19

Reply:
it shouldn't be null. The backend omits in that case otherwise everything would be either

Reply:
right, we coerce `null` to `undefined` when processing the API response 
https://github.com/yldio/asap-hub/blob/e7202ba33eede1e7b391c6c43ce02b298909de27/apps/asap-server/src/entities/user.ts#L97